platform/podjs/package.json

```diff
@@ -12,15 +12,18 @@
   "scripts": {
     "build": "shx rm -rf tsconfig.tsbuildinfo dist && tsc --emitDeclarationOnly && rollup -c",
     "build-downstream": "../../build.js --start @polypoly-eu/podjs",
-    "lint": "eslint src --ext .ts"
+    "lint": "eslint src --ext .ts",
+    "test": "jest"
   },
   "dependencies": {
     "@polypoly-eu/fetch-spec": "file:../feature-api/api/fetch-spec",
-    "@polypoly-eu/manifest-parser": "file:../feature-api/api/manifest-parser",
     "@polypoly-eu/pod-api": "file:../feature-api/api/pod-api",
     "@polypoly-eu/rdf": "file:../feature-api/api/rdf",
     "@zip.js/zip.js": "2.3.7",
-    "rdf-js": "^4.0.2"
+    "rdf-js": "^4.0.2",
+    "@types/semver": "^7.3.3",
+    "fp-ts": "^2.8.2",
+    "io-ts": "2.2.10"
```

Comment:
This version is simply the last one existing when this was created. Current one is 2.2.16. Can you upgrade to this one, if possible?